tests/python/unittest/test_gluon_estimator.py

```diff
@@ -394,24 +388,6 @@ def test_val_net():
                     val_loss=val_loss,
                     val_net=val_net)
 
-    with pytest.raises(RuntimeError):
-        est.fit(train_data=dataloader,
-                val_data=dataloader,
-                epochs=num_epochs)
-
-    ''' test weight sharing of sequential networks with namescope '''
```

Comment:
Actually test for new method of weight sharing exists. Originally, there are two tests: "test weight sharing of sequential networks with namescope" and "test weight sharing of sequential networks without namescope". Since in new API, we don't have to define children block inside "with net.name_scope()", I remove one of the tests. You can see in "def _get_test_network(params=None)", I maintain the original test for weight sharing.